clean_project/clean_file.py

```diff
@@ -0,0 +1,21 @@
+import unicodedata
+import sys
+
+table = dict.fromkeys(i for i in xrange(sys.maxunicode)
+                      if unicodedata.category(unichr(i)).startswith('P'))
+
+
+def remove_punctuations(line):
+    a = line.decode('utf-8').translate(table)
+    return a
+
+
+def clean_up(src_path, dest_path):
+    with open(src_path) as source_file:
```

Comment:
you can probably open a file both for read and write as a utf-8 text, hence avoiding the `.decode` above. have a look at the `codecs` package from the standard library.